R/tm_missing_data.R

```diff
@@ -121,14 +133,28 @@ tm_missing_data <- function(label = "Missing data",
 
   checkmate::assert_multi_class(pre_output, c("shiny.tag", "shiny.tag.list", "html"), null.ok = TRUE)
   checkmate::assert_multi_class(post_output, c("shiny.tag", "shiny.tag.list", "html"), null.ok = TRUE)
+
+  available_decorators <- c("summary_plot", "summary_plot", "combination_plot", "by_subject_plot", "summary_table")
+  if (checkmate::test_list(decorators, "teal_transform_module", null.ok = TRUE)) {
+    decorators <- if (checkmate::test_names(names(decorators), subset.of = c("default", available_decorators))) {
+      lapply(decorators, list)
+    } else {
+      list(default = decorators)
+    }
+  }
```

Comment:
Conversion of `decorators` object if it is a flat list of `teal_transform_module`, it will preserve current behavior.